src/item.cpp

```diff
@@ -4740,6 +4740,13 @@ bool item::goes_bad() const
     return is_food() && get_comestible()->spoils != 0_turns;
 }
 
+bool item::goes_bad_after_opening() const
+{
+    // returns true if this item goes bad, or if this item is a preserving container and its contents can go bad.
```

Comment:
There is already documentation for this in the header. Why repeat this here? And why repeat the code as comment? The comment doesn't tell more than the code itself already does.

Reply:
Fixed.